lib/nfs_v3.c

```diff
@@ -4542,7 +4550,6 @@ nfs3_pread_async_internal(struct nfs_context *nfs, struct nfsfh *nfsfh,
                 return -1;
         }
 
-        pdu->requested_read_count = count;
```

Comment:
Comment:
rpc_nfs3_read_task must already have done this, and moreover it's pointless to do it later as the call can already have been issued.